src/nft/components/bag/BagFooter.tsx

```diff
@@ -1,16 +1,42 @@
 import { BigNumber } from '@ethersproject/bignumber'
+import { Trans } from '@lingui/macro'
 import Loader from 'components/Loader'
 import { Box } from 'nft/components/Box'
 import { Column, Row } from 'nft/components/Flex'
 import { bodySmall } from 'nft/css/common.css'
 import { BagStatus } from 'nft/types'
 import { ethNumberStandardFormatter, formatWeiToDecimal } from 'nft/utils'
+import { AlertTriangle } from 'react-feather'
 import { useModalIsOpen, useToggleWalletModal } from 'state/application/hooks'
 import { ApplicationModal } from 'state/application/reducer'
+import styled from 'styled-components/macro'
 import { ThemedText } from 'theme'
 
 import * as styles from './BagFooter.css'
 
+const Footer = styled(Column)`
```

Comment:
also since we want to move off vanilla extract can we use `styled.div` instead and just add `display: flex` and `flex-direction: column`